server/data.py

```diff
@@ -219,6 +219,12 @@ def _count_gt(self, data, filters, chips, data_type):
             info_df = self.info_chip
             info_columns = self.info_columns_chip
 
+        # extract data freeze number from the data
+        try:
+            release_version = re.search(r'(?<=fgq.r)\d+', self.conf['sqlite_db']).group(0)
```

Comment:
Would it be possible to have release_version as a configuration item of its own? Then it would be clearer, more flexible and no surprises if the db file is named differently

Reply:
Sure, it makes more sense, I will update the code accordingly. Thank you for the review and comments!

Reply:
Updated